apps/colt-steele/src/lib/purchases.ts

```diff
@@ -3,7 +3,7 @@ import {Purchase} from '@skillrecordings/database'
 
 export async function updatePurchaseStatusForCharge(
   chargeId: string,
-  status: 'Valid' | 'Refunded' | 'Disputed' | 'Banned',
+  status: 'Valid' | 'Refunded' | 'Disputed' | 'Banned' | 'Restricted',
```

Comment:
`Restricted` is similar to `Valid` and gets added most places where `Valid` existed